sopel/modules/clock.py

```diff
@@ -157,27 +157,32 @@ def f_time_zone(bot, trigger):
 @module.commands('settz', 'settimezone')
 @module.example('.settz America/New_York')
 def update_user(bot, trigger):
+    """Set your preferred timezone.
+
+    Most timezones will work, but it's best to use one from
+    https://sopel.chat/tz
```

Comment:
I think there was a reason for using `<url>`. Per 07edfa4 it was to support auto-generating the plugin docs on the website. Something about bare URLs not getting linked due to how that page gets built…

Reply:
Meh. I'll revert that little change then.